pom.xml

```diff
@@ -220,6 +225,16 @@
 
   <dependencyManagement>
     <dependencies>
+      <dependency>
+        <groupId>commons-io</groupId>
+        <artifactId>commons-io</artifactId>
+        <version>2.6</version>
+      </dependency>
+      <dependency>
+        <groupId>org.apache.httpcomponents</groupId>
+        <artifactId>httpclient</artifactId>
+        <version>4.5.5</version>
```

Comment:
Ditto, or use the library plugin for this.

Reply:
I've added these deps here due to enforcer (Require upper bound dependencies error) and I am actually not sure how to apply excludes here. 
Could you assist with that or share some info on using library plugin?

Reply:
If you needed to do this, the POM mistakes are probably upstream in the `google-oauth` plugin.